Reconstruction/RecCalorimeter/options/runEcalReconstruction.py

```diff
@@ -0,0 +1,60 @@
+EVTMAX=100
+
+from Gaudi.Configuration import *
+
+from Configurables import ApplicationMgr, FCCDataSvc, PodioOutput
+
+podioevent   = FCCDataSvc("EventDataSvc")
+
+# reads HepMC text file and write the HepMC::GenEvent to the data service
+from Configurables import PodioInput
+podioinput = PodioInput("PodioReader", filename="/afs/cern.ch/user/n/novaj/public/FCCSW_tests/output_ecal.root", collections=["ECalHits", "ECalClusters", "GenParticles", "GenVertices"], OutputLevel=DEBUG)
```

Comment:
Can you give some more information on the segmentation violation?

So far we have put test files (that is, files that are small and only for testing purposes) into our afs space. How many events are in / how big is the file?